resources/keybinds.txt

```diff
@@ -29,20 +29,23 @@ KEYBINDINGS:
 [TOGGLE_STOCKS:KEY_I]
 [TOGGLE_ZONES:KEY_U]
 [TOGGLE_OCCLUSION:KEY_O]
+[TOGGLE_FOG:KEYS_SEMICOLON]
 [TOGGLE_CREATURE_MOODS:KEY_M]
 [TOGGLE_CREATURE_PROFS:KEY_P]
 [TOGGLE_CREATURE_JOBS:KEY_J]
 [TOGGLE_CREATURE_NAMES:KEY_N]
 [CHOP_WALLS:KEY_C]
 [CYCLE_TRACKING_MODE:KEY_F]
 [RESET_VIEW_OFFSET:KEY_Z]
-[DECR_SEGMENT_Z:KEY_1]
-[INCR_SEGMENT_Z:KEY_2]
+[DECR_SEGMENT_Z:KEYPAD_3]
+[INCR_SEGMENT_Z:KEYPAD_9]
```

Comment:
as in https://github.com/DFHack/stonesense/pull/132 I suggest you keep the Z keybindings as they are